source/databases-collections.txt

```diff
@@ -103,6 +103,40 @@ document in |vscode-short| and view its contents.
 2. Click :guilabel:`Delete Document`.
 3. Click :guilabel:`Yes` to confirm.
 
+*To insert a document*, you can: 
+
+1. Right-click the collection where you want to insert a document.
+#. Click :guilabel:`Insert Document`.
+
+   |vsce| opens the Playground with a template for adding a document to
+   the collection. 
+
+   .. code-block:: json
+      :copyable: false 
+      :caption: Example Playground Template 
+      :linenos: 
+      :emphasize-lines: 8-10
+
+      // MongoDB Playground
+      // Use Ctrl+Space inside a snippet or a string literal to trigger completions.
+
+      // The current database to use.
+      use('<database-name>');
+
+      // Create a new document in the collection.
+      db.getCollection('<collection-name>').insertOne({
+
+      });
+
+#. Paste the document to add inside the
+   :method:`db.collection.insertOne()` method and click the
+   :guilabel:`Play Button`. 
```

Comment:
[suggestion] a few suggestions to consider for clarity!

```suggestion
#. Paste the document you want to add inside the
   :method:`db.collection.insertOne()` curly braces and click the
   :guilabel:`Play Button`. 
```

Reply:
I think usage of "method" is correct here.